### PR TITLE
Added exotel as a vendor for sending sms  : India Only

### PR DESCRIPTION
**Pull request by sarathsp06**
Testing ? - Will do later

### CODE REVIEW

File: config.go [hunk 1]
```diff
@@ -19,6 +19,10 @@ var config struct {
 		MessageBird struct {
 			AccessKey string `yaml:"access_key"`
 		}
+		Exotel struct {
+			AccountSID string `yaml:"sid"`
```
Comment (samwierema) [inline config.go]: Can you change this to `yaml:"account_sid"`, or even to `SID string`, so it's consistent?

File: exotel.go [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+}
+
+//NewExotel creates a new
+func NewExotel(sid ,token string) *Exotel {
+	Exotel := &Exotel{requester: gorequest.New().SetBasicAuth(sid, token), AccountSid: sid, Token: token}
+	return Exotel
+}
+
+//SetHeaders Sets needed headers
+func (c *Exotel) setHeaders() *Exotel {
```
Comment (samwierema) [inline exotel.go]: I think you can just include this in the `sendSMS()` function

File: exotel.go [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+		Set("User-Agent", "sheetblast-v1").
+		Set("Accept-language", "es").
+		Set("Content-Type", "application/x-www-form-urlencoded").
+		Set("accept", "application/json")
+	return c
+}
+
+//sendSMS sends sms  ,obviously  
+//This API is rate limited to 200 SMSes per minute. Once this limit has been crossed, your requests will be rejected with an HTTP 429 'Too Many Requests' code.
+func (c *Exotel) sendSMS(from string, to []string,body string) ( err error) {
```
Comment (samwierema) [inline exotel.go]: I think you can just merge `Send` and `sendSMS`, there's no reason to keep them separated.

**Comment by marcelcorso** [comment]
hey @sarathsp06 wanna give some <3 to this pull request? its almost there!

**Comment by sarathsp06** [comment]
Sure I will update the pull request today

**Comment by sarathsp06** [comment]
Got busy with work will try to look at it today :D

**Comment by sarathsp06** [comment]
Updated pull request

**Comment by sarathsp06** [comment]
@marcelcorso please review the request

**Comment by samwierema** [comment]
@sarathsp06 nice update! 

@marcelcorso LGTM

**Pull request merged**